### DISCUSSION TITLE
Error in read.dcf(path) : cannot open the connection

### DISCUSSION THREAD

**Discussion by hirenpatel35** [community/community#26035] [Actions]
<p>We have R code in Github repository and our purpose is GitHub actions check automatically R code using github action. So we are using all these action from R-lib actions (<a href="https://github.com/r-lib/actions/tree/master/examples" rel="nofollow noopener">https://github.com/r-lib/actions/tree/master/examples</a>) into our repository but getting same error "read.dcf(path) : cannot open the connection " with almost all actions are available in R-lib (<a href="https://github.com/r-lib/actions/tree/master/examples" rel="nofollow noopener">https://github.com/r-lib/actions/tree/master/examples</a>). For error see below image as reference. So is there permission issue with github actions?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/772f6e164ef3c4071ec12df390d72bda6e877c22.png" data-download-href="/uploads/short-url/h0mr7aDWdggHazXe4SJXpJdQ9DY.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3881201/181094896-e618e100-8d9f-42d8-b884-34636f5528c4.png" alt="image" data-base62-sha1="h0mr7aDWdggHazXe4SJXpJdQ9DY" width="690" height="389" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/772f6e164ef3c4071ec12df390d72bda6e877c22_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">961×542 29.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>We would appreciate your help.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/hirenpatel35">@hirenpatel35</a>,</p>
<p>This is due to ‘DESCRIPTION’ file doesn’t exist in your repository.</p>
<p>I can repro the same error and it’s fixed after i add the file.</p>
<p>Please check my workflow for your reference.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/weide-zhou/RLint/runs/1049391411?check_suite_focus=true#step:4:20" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/52772761/181094897-62652e0c-475b-44e7-bc9b-7e7c0a741d74.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/weide-zhou/RLint/runs/1049391411?check_suite_focus=true#step:4:20" target="_blank" rel="nofollow noopener">weide-zhou/RLint</a></h3>

<p>Contribute to weide-zhou/RLint development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thanks</p>

**Comment by hirenpatel35** [comment]
<p>Thank You for this, Now it’s working</p>

**Comment by weide-zhou** [comment]
<p>Glad to know it works! <a class="mention" href="/hirenpatel35">@hirenpatel35</a></p>
<p>It’s recommended to mark the answer then it could help others who encounter the same problem.</p>
<p>Thanks</p>

**Comment by EvertonTLima** [comment]
Não aparece mais o fluxo, estou com o mesmo erro. Se puder compartilhar
